examples/nlp/bert_pretraining.py

```diff
@@ -113,7 +114,8 @@
 parser.add_argument("--only_mlm_loss", action="store_true",
                     default=False, help="use only masked language model loss")
 parser.add_argument("--total_iterations_per_gpu", default=-1,
```

Comment:
do you have use cases for this?

Reply:
yes, but "total_iterations_per_gpu" is a very confusing name. Why just not name this "max_steps"?